intellij-plugin/src/main/kotlin/com/apollographql/ijplugin/refactoring/migration/v3tov4/ApolloV3ToV4MigrationProcessor.kt

```diff
@@ -24,12 +24,14 @@ import com.apollographql.ijplugin.refactoring.migration.v3tov4.item.UpdateThrowA
 import com.apollographql.ijplugin.refactoring.migration.v3tov4.item.UpdateWebSocketReconnectWhen
 import com.intellij.openapi.project.Project
 
-private const val apollo4LatestVersion = "4.0.0-beta.4"
-
 /**
  * Migrations of Apollo Kotlin v3 to v4.
  */
 class ApolloV3ToV4MigrationProcessor(project: Project) : ApolloMigrationRefactoringProcessor(project) {
+  companion object {
+    const val apollo4LatestVersion = "4.0.0-beta.4"
```

Comment:
You could use `com.apollographql.apollo3.compiler.APOLLO_VERSION` to have this match the published version automatically without having to bump this every time